docs/tutorial.rst

```diff
@@ -322,3 +327,103 @@ decorator:
+would fire 10,000 times for every updated row even though the query above
+is only issuing one single update statement.
+
+Although triggers are issued at the database level and will not induce
+expensive round trips to the database, it can still be unnecessarily expensive
+to do row-level triggers for certain situations.
+
+Statement-level triggers, in contrast to row-level triggers, are executed
+once per statement. One only needs to provide ``level=pgtrigger.Statement`` to
+the trigger to configure this. However,
+keep in mind that trigger conditions and are largely not applicable to
+statement-level triggers since the ``OLD`` and ``NEW`` row variables are
+always ``NULL``.
+
+Postgres10 introduced the notion of "transition tables"
+to allow users to access old and new rows in a statement-level trigger
+(see `this <https://dba.stackexchange.com/a/177468>`__ for an example).
+One can use the `pgtrigger.Referencing` construct to write a statement-level trigger
+that references the old and new rows.
+
+For example, imagine we have a log model that logs changes to a table
+and keeps track of an old field and new field for every update.
+We can create a statement-level trigger that logs the old and new
+fields from a transition table to this persisted log model like so:
+
+.. code-block:: python
+
+    from django.db import models
+    import pgtrigger
+
+
+    class LogModel(models.Model):
+        old_field = models.CharField(max_length=32)
+        new_field = models.CharField(max_length=32)
+
+
+    @pgtrigger.register(
+        pgtrigger.Trigger(
+            level=pgtrigger.Statement,
+            when=pgtrigger.After,
+            operation=pgtrigger.Update,
+            referencing=pgtrigger.Referencing(old='old_values', new='new_values'),
+            func=f'''
+                INSERT INTO {LogModel._meta.db_table}(level, old_field, new_field)
+                SELECT
+                    'STATEMENT' AS level,
+                    old_values.field AS old_field,
+                    new_values.field AS new_field
+                FROM old_values
+                    JOIN new_values ON old_values.id = new_values.id;
+                RETURN NULL;
+            ''',
+        )
+    )
+    class LoggedModel(models.Model):
+        field = models.CharField(max_length=32)
+
+
+With this trigger definition, we'll now have the following happen with only
+one additional query in the trigger:
+
+.. code-block:: python
+
+    LoggedModel.objects.bulk_create([LoggedModel(field='old'), LoggedModel(field='old')])
+
+    # Update all fields to "new"
+    LoggedModel.objects.update(field='new')
+
+    # The trigger should have persisted these updates
+    print(LogModel.values('old_field', 'new_field'))
+
+    >>> [{
+      'old_field': 'old',
+      'new_field': 'new'
+    }, {
+      'old_field': 'old',
+      'new_field': 'new'
+    }]
+
+.. note::
+
+    Check out `django-pghistory <https://pghistory.readthedocs.io>`__
```

Comment:
This link isn't working for me.

Reply:
@ryanhiebert good catch! It's `django-pghistory`. @tomage let me update this before you merge